src/screenshareviewfacade/ScreenShareViewFacade.ts

```diff
@@ -18,12 +18,12 @@ export default interface ScreenShareViewFacade {
    * Starts viewing the screen share within an HTML element. Note that an
    * HTMLCanvas will be placed inside of this element.
    */
-  start(element: HTMLDivElement): void;
+  start(element: HTMLDivElement): Promise<void>;
```

Comment:
I think this is a breaking behavior change and would require at least minor SDK version bump.

Reply:
Versions are bumped when we call `script/publish`. You do not need to adjust the version as part of the PR. The next time we `npm publish`, we will run `script/publish` and can bump the minor version.